### PR TITLE
Improve coverage owschecker

### PR DESCRIPTION
**Pull request by mariavkourouni**

### CODE REVIEW

**Comment by loicgasser** [comment]
jenkins test

File: chsdi/tests/integration/test_owschecker.py [hunk 1]
```diff
@@ -15,6 +17,25 @@ def tearDown(self):
         super(TestOwsChecker, self).tearDown()
         del self.capture
 
+    def _callFUT(self):
+        base_url = 'http://wms.geo.admin.ch'
```
Comment (loicgasser) [inline chsdi/tests/integration/test_owschecker.py]: This variable should be red from production.ini file.

Comment (mariavkourouni) [inline chsdi/tests/integration/test_owschecker.py]: What do you mean? should I add at production.ini something like: base_url = ${base_url}
or
use here something like this: base_url = "http://" + app.registry.settings['wmshost'] + '/'

Comment (loicgasser) [inline chsdi/tests/integration/test_owschecker.py]: Second

**Pull request merged**